src/cpp/dpm_sampler.cpp

```diff
@@ -1425,16 +1457,16 @@ ssret* sample_class(unsigned int i, unsigned int step)
             nodist = cn->rd1[i];
 
             if (b1 != 1.0) {  // theta < 1.0
-                log_P[st] = gsl_sf_log((double)tw);
-                log_P[st] += nodist * gsl_sf_log(b1);
-                log_P[st] += dist * gsl_sf_log(b2);
+                log_P[st] = std::log((double)tw);
```

Comment:
`std::log<double>(tw)`

Reply:
Doesn't work. Compiler then looks for `std::log<double>(unsigned int &)`